build.sh

```diff
@@ -84,5 +85,10 @@ for target in "$@"; do
       fi
       ;;
 
+    it)
+      doBuild jib-core  ./gradlew $gradleOptions integrationTest
+      doBuild jib-maven-plugin   ./mvnw $mavenOptions integration-test -U
```

Comment:
To run the Maven integration test, we actually have it be in a profile: `./mvnw $mavenOptions verify -Pintegration-tests`